testing/sdk_utils.py

```diff
@@ -25,18 +25,30 @@ def get_service_name(default: str) -> str:
     return os.environ.get("INTEGRATION_TEST__SERVICE_NAME") or default
 
 
-def list_reserved_resources():
-    '''Displays the currently reserved resources on all agents via state.json;
-       Currently for INFINITY-1881 where we believe uninstall may not be
-       always doing its job correctly.'''
-    state_json_slaveinfo = dcos.mesos.DCOSClient().get_state_summary()['slaves']
+def list_reserved_resources(service_name: str) -> bool:
```

Comment:
If we only call this on uninstall and with the service name, we could change the name of the function.